### PR TITLE
Main Dev branch changes

### PR DESCRIPTION
**Pull request by cabrera-carlos**
Adding main sections and placeholders for text.

### CODE REVIEW

File: src/locales/en.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
-    
+    "certs-heading" : "Certificated by",
```
Comment (cabrera-carlos) [inline src/locales/en.json:2]: Certified

File: src/locales/es.json [hunk 2]
```diff
@@ -1,12 +1,16 @@
 {
+    "certs-heading" : "Certificado por",
+
     "app.cta.text": "Contáctanos hoy",
-    "app.cta.submit" : "Enviar",
     "app.cta.phone" : "Teléfono*",
     "app.cta.name" : "Nombre *",
     "app.cta.state" : "Estado *",
     "app.cta.interested" : "Esto interesado en *",
```
Comment (cabrera-carlos) [inline src/locales/es.json:8]: Estoy

File: src/locales/en.json [hunk 3]
```diff
@@ -9,5 +10,7 @@
     "app.cta.inmigration" : "Inmigration",
```
Comment (cabrera-carlos) [inline src/locales/en.json:10]: Immigration

Comment (brauliobolano) [reply src/locales/en.json:10]: Thanks for feedback

**Review by brauliobolano — APPROVED** [review]

**Review by brauliobolano — APPROVED** [review]

**Review by brauliobolano — APPROVED** [review]

**Review by brauliobolano — APPROVED** [review]

**Pull request merged**